src/client/core/utils/event.js

```diff
@@ -67,7 +75,10 @@ export function documentReady () {
             nativeMethods.setTimeout.call(window, onContentLoaded, 1);
         else {
             bind(document, 'DOMContentLoaded', onContentLoaded);
-            bind(window, 'load', ready);
+            bind(window, 'load', () => {
```

Comment:
I don't understand it. We will have ready called from `DOMContentLoaded` already (which always triggered before `window.onload`. I thought the algorithm will be:
1. Wait for `DOMContentLoaded`
2. Wait for `window.load` or `pageLoadTimeout` whichever comes first.